usermods/Tubes/node.h

```diff
@@ -295,11 +298,10 @@ class LightNode {
         Serial.println();
 #endif
 
-        __attribute__((unused)) auto err = espnowBroadcast.send((const uint8_t*)message, sizeof(*message));
-
+        __attribute__((unused)) auto success = espnowBroadcast.send((const uint8_t*)message, sizeof(*message));
```

Comment:
api was changed to return a bool, but this wasn't updated appropriately, so code was falsely printing failed